Source/WebCore/platform/graphics/ColorSerialization.cpp

```diff
@@ -395,6 +398,9 @@ String serializationForRenderTreeAsText(const HSLA<float>& color, bool useColorF
 
 String serializationForCSS(const HWBA<float>& color, bool useColorFunctionSerialization)
 {
+    if (useColorFunctionSerialization)
+        return serializationForCSS(convertColor<ExtendedSRGBA<float>>(color), useColorFunctionSerialization);
+
     return serializationForCSS(convertColor<SRGBA<uint8_t>>(color), useColorFunctionSerialization);
```

Comment:
A little strange to be passing the boolean in for functions on both sides of the `if` statement. I’d be tempted to write `true` and `false`.